product/dialogs/miq_dialogs/miq_provision_openstack_dialogs_template.yaml

```diff
@@ -173,6 +182,15 @@
           :required_method: :validate_cloud_network
           :display: :edit
           :data_type: :integer
+        :network_port:
+          :values_from:
+            :method: :allowed_network_ports
+          :description: Network Port
+          :auto_select_single: false
+          :required: true
+          :required_method: :validate_network_port
```

Comment:
same here

Reply:
It exists in manageiq-providers-openstack (once the associated PR is pushed). This is an openstack-specific yaml file. Note that the same thing is done for validate_cloud_network already (only exists in the openstack provider), which was what I was following when implementing this.